velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -850,5 +854,19 @@ TEST_F(StringTest, findInSet) {
   EXPECT_EQ(
       findInSet("abåæçè", "\u002c\u0061\u0062\u00e5\u00e6\u00e7\u00e8"), 2);
 }
+
+TEST_F(StringTest, initCap) {
+  EXPECT_EQ(
+      initcap("àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþ"),
+      "Àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþ");
+  EXPECT_EQ(initcap("αβγδεζηθικλμνξοπρςστυφχψ"), "Αβγδεζηθικλμνξοπρςστυφχψ");
+  EXPECT_EQ(initcap("hello world"), "Hello World");
+  EXPECT_EQ(initcap("HELLO WORLD"), "Hello World");
```

Comment:
Please document this behavior in string.rst.

Reply:
I think that's what I was trying to show here https://github.com/facebookincubator/velox/pull/8676/files#diff-945c1e481b55ff702e669348c241946c1b7851d575913d1a0da54952750f96daR77

Reply:
I think it would be better to explicitly document this behavior, not just giving an example. Thanks!